### PR TITLE
Added destination column on networking requirements

### PR DESCRIPTION
**Pull request by rbrtbnfgl**
Added destination column on network requirements (https://github.com/rancher/rke2/issues/4691)

### CODE REVIEW

File: docs/install/requirements.md [hunk 1]
```diff
@@ -90,33 +90,34 @@ If you wish to utilize the metrics server, you will need to open port 10250 on e
...
-| TCP | 9098 | RKE2 server and agent nodes | Calico Typha health checks
-| TCP | 9099 | RKE2 server and agent nodes | Calico health checks
-| TCP | 5473 | RKE2 server and agent nodes | Calico CNI with Typha
-| UDP | 8472 | RKE2 server and agent nodes | Canal CNI with VXLAN
-| TCP | 9099 | RKE2 server and agent nodes | Canal CNI health checks
-| UDP | 51820 | RKE2 server and agent nodes | Canal CNI with WireGuard IPv4
-| UDP | 51821 | RKE2 server and agent nodes | Canal CNI with WireGuard IPv6/dual-stack
+| Protocol | Port        | Source            | Destination       | Description
+|----------|-------------|-------------------|-------------------|---|
+| TCP      | 9345        | RKE2 agent nodes  | RKE2 server nodes | Kubernetes API
```
Comment (brandond) [inline docs/install/requirements.md:95]: Might as well fix this while we're here.
```suggestion
| TCP      | 9345        | RKE2 agent nodes  | RKE2 server nodes | RKE2 supervisor API
```

**Review by brandond — CHANGES_REQUESTED** [review]
one nit on a port description; LGTM otherwise!

**Review by dereknola — APPROVED** [review]

**Review by manuelbuil — APPROVED** [review]

**Comment by rbrtbnfgl** [comment]
Yes after @manuelbuil merges his PR I'll rebase and fix mine.

**Review by brandond — APPROVED** [review]
lgtm pending rebase

**Pull request merged**